.github/workflows/labels.yml

```diff
@@ -0,0 +1,24 @@
+name: labels
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - .github/labels.yml
+      - .github/workflows/labels.yml
+  pull_request: # dry run only
+    paths:
+      - .github/labels.yml
+      - .github/workflows/labels.yml
+
+jobs:
+  labeler:
+    permissions:
+      contents: read
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: crazy-max/ghaction-github-labeler@v5
```

Comment:
We haven't typically used "less trusted" 3rd parties like this. What are the guarantees around which repository secrets are provided to jobs like this?

Reply:
https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions#about-secrets

> GitHub Actions can only read a secret if you explicitly include the secret in a workflow.

Seems like it should be fine.

Reply:
Yep 😉 It may not always had been the case, so good checking!
[crazy-max](https://github.com/crazy-max) does a lot of things in the docker/CI space, you may see his username around again!